felt/src/bigint_felt.rs

```diff
@@ -483,6 +483,9 @@ impl<const PH: u128, const PL: u128> Div for FeltBigInt<PH, PL> {
     // In Felts `x / y` needs to be expressed as `x * y^-1`
     #[allow(clippy::suspicious_arithmetic_impl)]
     fn div(self, rhs: Self) -> Self::Output {
+        if rhs.to_i8().unwrap() == 0 {
+            panic!("Can't divide Felt by zero")
```

Comment:
I think here we should throw an Error instead of panic. We can ask @Oppen what is the correct way to handle these things.

Reply:
I think it was mentioned in Slack, `panic!` is the only possible way to handle it in the operator itself, as it's expected to be chainable and thus can't return a `Result`. `checked_div` returns a `Result` for when things are expected to handle failure.
That said, this will crash for an unrelated reason: `to_i8().unwrap()` will crash for values over `127` or under `-128` (the latter should never happen tho). Use `.is_zero()` instead.

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs/pull/817/commits/0d286f661b3b9a74eaf64b875925b7f4ee03fa6e